src/components/collapsible/Collapsible.js

```diff
@@ -107,7 +113,13 @@ class Collapsible extends React.PureComponent<Props, State> {
 
         return (
             <div className={sectionClassName}>
-                <div className={buttonClassName}>
+                <div
+                    className={buttonClassName}
+                    onBlur={onBlur}
+                    onFocus={onFocus}
+                    // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+                    tabIndex="0"
```

Comment:
Does the div need tabindex? Shouldn't it auto-tab into the PlainButton?

Reply:
you are right. its not required